src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/AbbreviationAsWordInNameCheck.java

```diff
@@ -348,13 +348,11 @@ private static boolean isInterfaceDeclaration(DetailAST variableDefAst) {
     private static boolean hasOverrideAnnotation(DetailAST methodModifiersAST) {
         boolean result = false;
         for (DetailAST child : getChildren(methodModifiersAST)) {
-            if (child.getType() == TokenTypes.ANNOTATION) {
```

Comment:
We are specifically looking for the annotation override and only want to examine annotations in a list of children from the modifiers. Since modifiers can only contain annotations and modifiers, it is not possible to have a non-annotation with the text `Override`, so the check isn't necessary.
Another case of preventing doing more work then necessary.